src/main/java/com/google/gfx/model/User.java

```diff
@@ -0,0 +1,28 @@
+package com.google.gfx.model;
+
+/**
+ * A user of Google Foreign Exchange.
+ */
+public class User {
+
+  /**
+   * User's email address, which is also their key.
+   */
+  private String email;
+
+  /**
+   * User's name.
+   */
+  private String name;
+
+  /**
+   * Where the user is located.
+   */
+  private Location location;
```

Comment:
I think we also talked about adding a "home currency", but that can actually live in the `Location` class.